gpflow/models/gplvm.py

```diff
@@ -144,14 +145,18 @@ def __init__(
         assert self.X_prior_var.shape[0] == self.num_data
         assert self.X_prior_var.shape[1] == self.num_latent_gps
 
-    def log_likelihood(self) -> tf.Tensor:
+    def maximum_log_likelihood_objective(self) -> tf.Tensor:
```

Comment:
these need updating right?
```suggestion
    def _training_loss(self) -> tf.Tensor:
```